packages/components/src/components/files/FileTree.tsx

```diff
@@ -104,12 +105,13 @@ const nodeIsEmpty = (id: string): boolean => {
 };
 
 const Header = props => {
-    const { style, onSelect, node, customStyles, checked, handleCheckbox, useFileIconCls } = props;
+    const { style, onSelect, node, customStyles, checked, handleCheckbox, useFileIconCls, mvDirMode } = props;
     const isDirectory = node.children !== undefined;
     const icon = isDirectory ? (node.toggled ? faFolderOpen : faFolder) : faFileAlt;
 
+    const emptyDirectoryText = mvDirMode ? 'Move to this directory' : 'No Files Found';
```

Comment:
This `mvDirMode` prop and the text here seem pretty specific to the moduleEditor implementation. I would suggest instead we add an options `emptyDirectoryText` prop which defaults to "No Files Found" and then the move dir case can pass in the alternate text.